### DISCUSSION TITLE
Issue in Android App

### DISCUSSION THREAD

**Discussion by cawa-93** [community/community#3934] [Mobile] [answered]
When i Try open My repo:
https://github.com/cawa-93/vite-electron-builder
![зображення](https://user-images.githubusercontent.com/1662812/117786680-4c6aa380-b235-11eb-9049-07f7ddd6c8dd.png)

**Comment by Iron-Ham** [comment, marked as answer]
Thank you for reporting this. I've created an issue and we'll begin tracking this internally.
